webhook/resources/backup/mutator.go

```diff
@@ -50,6 +53,35 @@ func (b *backupMutator) Create(request *admission.Request, newObj runtime.Object
 	}
 	patchOps = append(patchOps, patchOp)
 
+	backupLabels := make(map[string]string)
```

Comment:
The newly created `backupLabels` will override the existing labels.
Please refer to https://github.com/longhorn/longhorn-manager/blob/master/webhook/resources/volume/mutator.go#L150.

Reply:
Since I save the label into backup, therefore, I think it maybe won't override the label of volume.

Reply:
I mean the labels which are set to the backup creation request before the mutation.

Reply:
Sorry for misunderstanding you. But I will save the label under `/spec/labels` rather than `/metadata/labels`. 

Do you mean I need to save the value into `/metadata/labels`?

Reply:
No. I mean `/spec/labels`. The link I mentioned is a pattern for your reference.
Also refer to https://github.com/longhorn/longhorn-manager/pull/1486#discussion_r962114328

Reply:
Got it. I will try to follow the pattern to implement. Thanks.

Reply:
Follow the suggestion modify `backupLabels` initialize patter to avoid overwrite existing value.

https://github.com/longhorn/longhorn-manager/blob/b91976a861a4ef034334b6a3f7d56a44185aa1b5/webhook/resources/backup/mutator.go#L60-L63